<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trương Võ Trường Thy - K244060809</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidgetSong">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>581</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Song ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Song Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Singer</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>S1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Chốn Sa Mạc</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Lam</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>P2</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>Nếu ngày ấy</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Soobin</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>P3</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Cháu lên ba</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>Xuân Mai</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>P4</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>Baby</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>Justin</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>P5</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>Enchanted</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>Taylor</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Song ID: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Song Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Singer: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSongID">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSongName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSinger">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
